### ISSUE TITLE
Missing .gitignore

### ISSUE THREAD

**Issue by ohle** [erosb/generator-typescript-backbone-amd#4]
The `.gitignore` file seems to be missing from the template after installation. I'm getting the following error:
```
events.js:85
      throw er; // Unhandled 'error' event
            ^
AssertionError: Trying to copy from a source that does not exist: /home/ohle/lib/node_modules/generator-typescript-backbone-amd/app/templates/.gitignore
```
when scaffolding a new project.

Might be related to https://github.com/yeoman/generator-polymer/issues/240.

**Comment by erosb** [comment]
Hi @ohle !

Thanks for reporting it. I couldn't reproduce the issue (the generator ran without any error and the .gitignore file exists). What version of yeoman are you using?

**Comment by ohle** [comment]
I'm using 1.4.5

**Comment by erosb** [comment]
Weird, it worked for me both with 1.4.8 and 1.5.1. I will try it with a fresh install

**Comment by erosb** [comment]
@ohle I've just released 0.0.3 on npm, could you please test it?

Thanks

**Comment by ohle** [comment]
Works now. Thank you!
